release-notes.txt

```diff
@@ -0,0 +1,222 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-14
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - fixed bug related to terminaton MAC entries not being correctly removed on IP deletion.
+  - resolved STP issue where the initial STP state was missed, causing ports to forward/flood instead of blocking.
+  - added initial support for Broadcom KNET interfaces.
+- Linux kernel updated to version 5.15.22
+
```

Comment:
shouldnt there also be something about the STP on bonded interfaces? maybe under bug fixes?

Reply:
Moved to separate MR

Reply:
Moved to separate MR